exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -120,7 +120,7 @@ drill.exec: {
     affinity.factor: 1.2
   },
   sys.store.provider: {
-    class: "org.apache.drill.exec.store.sys.zk.ZkPStoreProvider",
+    class: "org.apache.drill.exec.store.sys.store.provider.ZookeeperPersistentStoreProvider",
```

Comment:
Sure. Overriding config with what is already a default value(ZkPStoreProvider) seems odd though. Back to your point, this patch proposes a major design/API change around old E/PStore so all references to ZkPStoreProvider will fail following this patch.